CHANGES_NEXT_RELEASE

```diff
@@ -2,4 +2,18 @@ Add protocol in device provision if not provided (#652)
 Fix: timestamp is now included just in non empty (more than id and type) multienty entitites
 Fix: object_id fields are not introduced in CB requests by alias plugin (#660)
 Add: uses provisioned timezone to generate TimeInstant values (#672)
-Using precise dependencies (~=) in packages.json
\ No newline at end of file
+Using precise dependencies (~=) in packages.json
+Fix: updating dependencies due to known vulnerabilities in the previous ones
+	async: 1.5.2 -> 2.6.1
+	body-parser: ~1.11.0 -> ~1.18.3
+    express: ~4.11.2 -> ~4.16.4
+    jison: 0.4.17 -> 0.4.18
+    logops: 1.0.0 -> 2.1.0
```

Comment:
I understand the line about logops should be removed, as this dependency has not changed version at the end.